gcloud/translate/__init__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2015 Google Inc. All rights reserved.
```

Comment:
Yup. It's coming with the tests. I wanted to get the bones out for comments before committing to an API.